docs/source/nodes_and_pipelines/modular_pipelines.md

```diff
@@ -1,53 +1,23 @@
 # Modular pipelines
 
-## What are modular pipelines?
+In many typical Kedro projects, a single (“main”) pipeline increases in complexity as the project evolves. To keep your project fit for purpose, we recommend you **separate your code into different pipelines (modules)** that are logically isolated and can be reused. Each pipeline should ideally be organised in its own folder, promoting easy copying and reuse within and between projects. Simply put: one pipeline, one folder.
 
-In many typical Kedro projects, a single (“main”) pipeline increases in complexity as the project evolves. To keep your project fit for purpose, we recommend that you create modular pipelines, which are logically isolated and can be reused. Modular pipelines are easier to develop, test and maintain, and are portable so they can be copied and reused between projects.
+Kedro supports this concept of modular pipelines with the following tools:
+- [How to create a new blank pipeline using the `kedro pipeline create` command](#how-to-create-a-new-blank-pipeline-using-the-kedro-pipeline-create-command)
+- [How to structure your pipeline creation](#how-to-structure-your-pipeline-creation)
+- [How to use custom new pipeline templates](#how-to-use-custom-new-pipeline-templates)
+- [How to share your pipelines](#how-to-share-your-pipelines)
 
-Modular pipelines allow you to instantiate pipelines multiple times, while allowing the user to override inputs/outputs/parameters. They are reusable within the same codebase, and shareable across projects via [micro-packaging](micro_packaging.md). This is the modern way to use Kedro, and will change the way you think about your pipelines.
+## How to create a new blank pipeline using the `kedro pipeline create` command
```

Comment:
Long subheads like this look very awkward in the left hand table of contents. Is it necessary to have the full title or could it just be "How to create a new blank pipeline" ?